dump/mapping/bufr_amsua.py

```diff
@@ -71,90 +204,8 @@ def _re_map_variable(self, comm, container):
             ta = container.get('variables/brightnessTemperature', sat_id)
```

Comment:
The `variables/brightnessTemperature` can be simplified to `brightnessTemperature`